src/main/java/com/opencart/steps/HeaderPageBL.java

```diff
@@ -19,4 +28,66 @@ public RegisterPageBL clickOnRegisterButton() {
         headerPage.getRegisterButton().click();
         return new RegisterPageBL();
     }
+
+    public HeaderPageBL availableProductOptions() {
+        AddToShoppingCartModel addToShoppingCartModel = AddToShoppingCartModelRepository.getAddToShoppingCartModel();
+        inputText(addToShoppingCartModel.getTextInput());
+        inputAreaText(addToShoppingCartModel.getTextAreaInput());
+        radioButton();
+        checkboxButton();
+        sliderButton();
+        uploadFileButton();
+        addToCartButton();
```

Comment:
Could you please, specify actions you perform in each method name? For example, clickOnRadioButton